src/EnergyPlus/OutputProcessor.cc

```diff
@@ -6685,8 +6667,8 @@ void SetInitialMeterReportingAndOutputNames(EnergyPlusData &state,
                 if (op->EnergyMeters(WhichMeter).RptTS) {
                     ShowWarningError(state,
                                      "Output:Meter:MeterFileOnly requested for \"" + op->EnergyMeters(WhichMeter).Name +
-                                         R"(" (TimeStep), already on "Output:Meter". Will report to both )" + state.files.eso.filePath.string() +
-                                         " and " + state.files.mtr.filePath.string());
+                                         R"(" (TimeStep), already on "Output:Meter". Will report to both )" +
+                                         state.files.eso.filePath.filename().string() + " and " + state.files.mtr.filePath.filename().string());
```

Comment:
Just get the `filename()` property from the file path.  For fully qualified `fs::paths` this will just return the filename.  For filename-only `fs::path` objects, this will just return the filename itself.  (https://godbolt.org/z/o8d8Mv56d)